boards/shared/Inc/Io/Io_SharedSpi.h

```diff
@@ -106,5 +106,5 @@ HAL_StatusTypeDef Io_SharedSpi_MultipleTransmitWithoutNssToggle(
  */
 HAL_StatusTypeDef Io_SharedSpi_TransmitWithoutNssToggle(
     const struct SharedSpi *spi,
-    uint8_t *               tx_data,
+    uint8_t *               tx_buffer,
```

Comment:
Should I be calling tx_data tx_buffer? @markomarkusi

Reply:
ya buffer sounds good!